docs/docs/02-concepts/01-preflight-and-inflight.md

```diff
@@ -28,7 +28,7 @@ Your preflight code runs once, at compile time, and defines your application's i
 
 For example, this code snippet defines a storage bucket using a class from the standard library:
 
-```js playground
+```js playground example
```

Comment:
Honestly Im wondering why we would not have all our examples openable in the `playground` and if that assumption is right, then maybe rather than add `example` I can key testing off `playground`

any thoughts here @Chriscbr @MarkMcCulloh

Reply:
Though.. maybe thats not the best approach atm, since this `example` keyword should eventually support library examples and typescript examples..

I may have talked myself out of this

Reply:
This looks alright to me 👍